content/departments/cloud/technical-docs/oidc_site_admin.md

```diff
@@ -1,31 +1,66 @@
+
+Use the slash command in Slack, type `/access_request` anywhere.
+
+Select the following and click `Send Request`.
+
+- **Request type**: `Permission set`
+- **Permissions set**: `Sourcegraph Cloud Org2Org`
+- **Permission duration**: `1 Hour`
+- **Add justification**: I need access to the cloud instance UI (Yes, you can just use canned response here)
+
+The request will be approved automatically. Once you have received confirmation from the `Entitle` slack app, process to the next step.
+
+Use the `/access_request` slash command on Slack again, and select the following
+
+- **Request type**: `Specific permission`
+- **Permissions set**: `Sourcegraph Cloud OKTA`
+- **Resource Type**: `groups`
+- **Resource**: Filter by the domain name of the cloud instance url, e.g. `company.sourcegraph.com`
+- **Role**: `Group Member`
+- **Grant Type**: `Direct`
+- **Permission duration**: `1 Hour`
+- **Add justification**: Please explain in detail why do you need access to the Cloud instance UI. It will be best to include relevant links to issues, slack thread to provide more context.
+
+The request will be routed to #cloud, #security, or your direct manager for approval. We will review the request and approve the access request.
+
+Please tag `@cloud-support` or `@security-support` in #cloud for immediate attention if it is time sensitive. If the request is related to an ongoing [incident](../../engineering/dev/process/incidents/index.md), please [page Cloud on-call engineer using OpsGenie](../../engineering/dev/process/incidents/index.md#incident-lead).
+
+- `/genie alert I need UI access to company.sourcegraph.com for cloud`
+
+Once the `Entitle` slack app has confirmed the permission is granted, you will have access to the instance for **1 hour**. Once it's expired, you will be logged-out and you will have to request access again from the beginning.
 
-Unless customer explicitly [disabled OIDC on Managed Instance](#disabling-okta-oidc-on-managed-instance)
+> Consider installing [requestly](https://requestly.io/) and use this [rule](https://app.requestly.io/rules/#sharedList/1670019946529-Michael-shared-list-12-2-2022) to automatically append the `sourcegraph-operator` query param on any instance signin page.
 
-2. All [Cloud Team members](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L1)
-3. Additonal Customer Engineeger(s) from [CE List](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L13) added to `config.yaml` via `additionalAdmins` - [sample](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/7e9066e537b02feb6013585d443fc27514b71a71/addepar/config.yaml#L5)
+To login to the Cloud instance UI, open `https://company.sourcegraph.com/sign-in?sourcegraph-operator`, and click `Continoue with Sourcegraph Operator` to authenticate.
 
 ## Disabling OKTA OIDC on Managed Instance
```

Comment:
I think this section should be now called "Disabling Sourcegraph management access", WDYT?